internal/storage/postgres/sensor/get.go

```diff
@@ -13,7 +13,7 @@ import (
 func (r *SensorRepository) GetAll(ctx context.Context) ([]*entities.Sensor, error) {
 	rows, err := r.store.GetAllSensors(ctx)
 	if err != nil {
-		return nil, err
+		return nil, r.store.HandleError(err)
```

Comment:
One thing I've learnt in Go so far is that it's always annoying to handle errors in one place. I would just return the error here and the person calling this function takes care of it. Maybe with `errors.Wrap()`. I'm also not happy with the error handling in the service layer...